app/grandchallenge/participants/models.py

```diff
@@ -54,3 +56,88 @@ def delete(self, *args, **kwargs):
 
     class Meta:
         unique_together = (("challenge", "user"),)
+
+
+def string_type_schema():
+    return {"type": "string"}
+
+
+class RegistrationQuestion(UUIDModel):
+    challenge = models.ForeignKey(
+        Challenge,
+        on_delete=models.CASCADE,
+    )
+
+    question_text = models.TextField(blank=False)
+    question_help_text = models.TextField(blank=True)
+
+    schema = models.JSONField(
+        default=string_type_schema,
+        help_text="A JSON schema definition against which an answer is validated. See https://json-schema.org/."
+        "Only Draft 7, 6, 4 or 3 are supported.",
+        validators=[JSONSchemaValidator()],
+    )
+
+    required = models.BooleanField(default=True)
+
+    @property
+    def has_answers(self):
+        return RegistrationQuestionAnswer.objects.filter(
+            question=self
+        ).exists()
+
+    class Meta:
+        unique_together = (("question_text", "challenge"),)
+
+    def clean(self):
+        super().clean()
+
+        if (
+            type(self)
+            .objects.filter(
+                challenge=self.challenge,
+                question_text=self.question_text,
+            )
+            .exclude(pk=self.pk)
+            .exists()
+        ):
+            raise ValidationError(
+                f"There is already an existing {self._meta.model._meta.verbose_name} with this question text"
+            )
+
+
+class RegistrationQuestionAnswer(models.Model):
```

Comment:
We don't have any answering views yet. Does it make sense to include this model in this PR? There is also no admin view for this. Given it is not used I would say to wait until you see how the integration goes.